theano/gpuarray/tests/test_dnn.py

```diff
@@ -656,55 +666,62 @@ def test_conv3d_none(self, border_mode, conv_mode):
                         border_mode,
                         conv_mode,
                         [(1, 1, 1), (2, 2, 2)],
+                        [(1, 1, 1)],
```

Comment:
About dilation problem for 3D conv, I am not familiar with concepts and purposes around dilation, but have you tried to either increase the size of image shape or decrease the size of kernel shape while applying a dilation other than (1, 1, 1) ? It seems it may works with dilation == (2, 2, 2) if you change shape of image value (at line 664), e.g: if `np.random.rand(10, 2, 6, 4, 11)` becomes `np.random.rand(10, 2, 7, 5, 11)`.

Could you try it ?